pyro/optim/adagrad_rmsprop.py

```diff
@@ -31,7 +33,7 @@ class AdagradRMSProp(Optimizer):
     :type delta: float
     """
 
-    def __init__(self, params, eta=1.0, delta=1.0e-16, t=0.1):
+    def __init__(self, params, eta: Optional[float] = 1.0, delta: Optional[float] = 1.0e-16, t: Optional[float] = 0.1):
```

Comment:
I think Optional is used for argument with [default None type](https://docs.python.org/3/library/typing.html#typing.Optional).

Reply:
Thanks for pointing it :), I think I need to remove Optional type hints where defaults are present.